runtime-modules/membership/src/lib.rs

```diff
@@ -145,10 +148,9 @@ pub trait Trait:
 pub(crate) const DEFAULT_MEMBER_INVITES_COUNT: u32 = 5;
 
 /// Public membership profile alias.
+pub type Membership<T> = MembershipObject<<T as frame_system::Config>::AccountId>;
 
-pub type Membership<T> = MembershipObject<<T as frame_system::Trait>::AccountId>;
-
-#[derive(Encode, PartialEq, Decode, Debug, Default)]
+#[derive(Encode, PartialEq, Decode, Debug, TypeInfo, Clone)]
```

Comment:
This is the only place in the code where `TypeInfo` is not the last derived trait, which could be slightly confusing:
```suggestion
#[derive(Encode, Decode, PartialEq, Debug, Clone, TypeInfo)]
```

Reply:
done